.github/workflows/azure-deploy-io-fast-login.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     uses: pagopa/dx/.github/workflows/function_app_deploy.yaml@edaa2a22738b738092e71dd9bdf6494aca593980
     with:
       # '-cd' suffix added inside the pipeline
-      environment: apps-prod
+      environment: app-prod
```

Comment:
A rebase is needed in order to align all workflows to newly created environment.
Can you do that?

Reply:
Nevermind, done it in  [align deploy pipelines to new environment](https://github.com/pagopa/io-auth-n-identity-domain/pull/192/commits/ecab157524db770c003fed8567b917944e805c83).